backend/src/render/html.mjs

```diff
@@ -92,17 +93,3 @@ const renderUsages = usages => `
     </tr>
   </table>
 `;
-
-/**
- * @param {UsageSummary} table
- * @returns {string}
- */
-const scyllaPricingUrl = (table) => {
-  const uri = new URL('https://www.scylladb.com/product/scylla-cloud/get-pricing?');
-  uri.searchParams.append('reads', Math.round(table.readThroughputBytes / table.averageItemSizeBytes).toString());
-  uri.searchParams.append('writes', Math.round(table.writeThroughputBytes / table.averageItemSizeBytes).toString());
-  uri.searchParams.append('itemSize', Math.round(table.averageItemSizeBytes / kB).toString());
-  uri.searchParams.append('storage', Math.round(table.sizeBytes / TB).toString());
-  uri.searchParams.append('cloudProvider', 'AWS')
-  return uri.href
-};
```

Comment:
Not problem with this particular change, but given I had to walk back through the code to find the caller function for `scyllaPricingUrl`, I noticed we are generating two different calls, one for on-demand and another provisioned.

On one hand, comparing on-demand vs our calculator should be spot-on.
On another hand, comparing provisioned vs our calculator is prone to show a different pricing.

Remember that the whole idea of Scylla is to basically group several tables under a single workload, so - by definition - if we provide 2 different links, there will be an increased pricing we'll show to the user considering they have both on-demand and provisioned tables.